ui/src/components/Footer.tsx

```diff
@@ -50,57 +41,70 @@ export default function Footer() {
   ];
 
   return (
-    <footer className="body-font sticky top-[100vh] text-gray-700 dark:text-gray-300">
-      <div className="mt-4 flex flex-col items-center px-8 py-4 sm:flex-row">
-        <div className="container mx-auto flex flex-col items-center space-x-4 sm:flex-row">
-          <p className="mt-4 text-xs text-gray-500 dark:text-gray-400 sm:mt-0">
-            <span className="hidden sm:inline">
-              {ref() && (
-                <>
-                  <a
-                    href={refURL()}
-                    className="text-violet-500 dark:text-violet-400"
-                  >
-                    {ref()}
-                  </a>
-                  &nbsp;|&nbsp;
-                </>
-              )}
-            </span>
-            <span className="block sm:inline">
-              &copy; {new Date().getFullYear()} Flipt Software Inc. All rights
-              reserved.
-            </span>
-          </p>
-          <p className="mt-4 text-xs text-gray-500 dark:text-gray-400 sm:mt-0">
-            <span className="hidden sm:inline">
-              <a
-                target="_blank"
-                rel="noreferrer"
-                href="https://docs.flipt.io/changelog/overview"
-                className="text-violet-500 dark:text-violet-400"
-              >
-                Changelog
-              </a>
-            </span>
-          </p>
+    <footer className="border-t border-gray-200 dark:border-gray-700 bg-white dark:bg-gray-900 text-gray-700 dark:text-gray-300">
```

Comment:
I'm not particularly fond of this new style. And these buttons 

![image](https://github.com/user-attachments/assets/c17f2e5d-940f-4e2f-9e9c-44beb3d59de2)

Reply:
I don't think I changed the buttons 🤔

Reply:
@erka i put back the footer the way it was before. I'll figure out another place to put the 'product' (oss/enterprise)